plugins/ltac2/tac2ffi.mli

```diff
@@ -175,7 +175,7 @@ val valexpr : valexpr repr
 val val_constr : EConstr.t Val.tag
 val val_ident : Id.t Val.tag
 val val_pattern : Pattern.constr_pattern Val.tag
-val val_preterm : Ltac_pretype.closed_glob_constr Val.tag
+val val_preterm : Glob_term.glob_constr Val.tag
```

Comment:
Do you really need this change? I know that in the current code this was always computed in an empty environment but I had in mind when I wrote this that we should be able to provide more primitive functions over preterms, such as extending the closure with other preterms.

Reply:
IMO this is more honest as the closure was always trivial. Once we add back nontrivial closures having the type change should also make it more likely that we catch where we need to handle them (currently since they're trivial we have no clue if we handle them correctly).

Reply:
If you insist I'll change it back